src/common/meta/src/key/flow/flow_info.rs

```diff
@@ -258,6 +281,38 @@ impl FlowInfoManager {
             TxnOpGetResponseSet::decode_with(TxnOpGetResponseSet::filter(key)),
         ))
     }
+
+    /// Update the last execution time of the flow.
+    pub async fn update_last_execution_time(
+        &self,
+        flow_id: FlowId,
+        last_execution_time: DateTime<Utc>,
+    ) -> Result<()> {
+        let prev_flow_info = self.get(flow_id).await?.unwrap();
```

Comment:
I am not sure if the `unwrap` is safe here.

Reply:
done, thank u for the comment